types/v1/metrics.go

```diff
@@ -49,26 +49,30 @@ type VolumeStats struct {
 	Name              string      `json:"Name"`
 }
 
+// VolStatus stores status of a volume
 type VolStatus struct {
 	Resource        Resource
 	ReplicaCounter  int64  `json:"replicacounter"`
 	RevisionCounter string `json:"revisioncounter"`
 }
 
+// Resource stores information about some resources
 type Resource struct {
 	Id      string            `json:"id,omitempty"`
 	Type    string            `json:"type,omitempty"`
 	Links   map[string]string `json:"links"`
 	Actions map[string]string `json:"actions"`
 }
 
+// Annotation contains information about iSCSI volume
 type Annotation struct {
 	IQN    string `json:"Iqn"`
 	Volume string `json:"Volume"`
 	Portal string `json:"Portal"`
 	Size   string `json:"Size"`
 }
 
+// StatsJSON stores the stastistics of a iSCSI volume
```

Comment:
` StatsJSON stores the statistic of an iSCSI volume`

Reply:
Just to add.
Please add a period at the end of comment.
This is what I personally prefer.